workspaces/lightspeed/plugins/lightspeed-backend/src/service/validation.ts

```diff
@@ -31,11 +31,12 @@ export const validateCompletionsRequest = (
       .json({ error: 'model is required and must be a non-empty string' });
   }
 
-  if (typeof reqData.provider !== 'string' || reqData.provider.trim() === '') {
-    return res
-      .status(400)
-      .json({ error: 'provider is required and must be a non-empty string' });
-  }
+  // Todo: // removing this validation for now, as we do not have provider information from the client.
```

Comment:
frontend should be able to read the list of available providers configured in RHDH plugin config.
as the visibility is set to frontend `@visibility frontend`

Reply:
Right, we can read the config from frontend too. Updated the code to send provider from Client.